### PR TITLE
Fix notification in M2K and MTA

### PR DESCRIPTION
**Pull request by masayag**
- Add topic for MTA
- Add hard-coded guest for M2K
- Update notification spec in MTA workflow

### CODE REVIEW

File: move2kube/m2k.sw.yml [hunk 1]
```diff
@@ -96,6 +96,8 @@ states:
             message: '"Please go to your <a href=\"" + $SECRET.move2kube_url + "/workspaces/"+ .workspaceId + "/projects/" + .projectId + "/outputs/"+ .transformId + "\">Move2Kube instance</a> and answers to questions in order to continue the Move2Kube workflow"'
             origin: "Move2Kube Workflow"
             topic: "Move2Kube Workflow"
+            targetUsers:
```
Comment (dmartinol) [inline move2kube/m2k.sw.yml:99]: I don't think it's needed, is it? I was removed from notifications API

File: mta/mta.sw.yaml [hunk 2]
```diff
@@ -150,8 +150,9 @@ states:
           refName: createNotification
           arguments:
             title: "\"MTA analysis report for \" + .application.repository.url"
-            message: '"<a href=\"" + $SECRET.mta_url+ "/hub/applications/" + (.application.id|tostring)+ "/bucket/" + .data.output + "\">MTA Analysis Rerport</a>"'
+            message: '"MTA Analysis Report:" + $SECRET.mta_url+ "/hub/applications/" + (.application.id|tostring)+ "/bucket/" + .data.output '
             origin: "MTA Analysis Workflow"
+            topic: "MTA Analysis"
             targetUsers:
```
Comment (dmartinol) [inline mta/mta.sw.yaml:156]: same: this can be removed IMO

**Review by dmartinol — COMMENTED** [review]
minor comments

File: move2kube/m2k.sw.yml (hunk 1 above)
Comment (masayag) [reply move2kube/m2k.sw.yml:99]: It depends on what is wanted:
with targetUsers the message goes to personal notification. without it it goes to updates.
in this context, it seems like a personal message, so having it here.

Comment (dmartinol) [reply move2kube/m2k.sw.yml:99]: I mean: targetUsers disappeared from notifications API, you cannot use it

Comment (dmartinol) [reply move2kube/m2k.sw.yml:99]: Removed from the latest commit

File: mta/mta.sw.yaml (hunk 2 above)
Comment (dmartinol) [reply mta/mta.sw.yaml]: Done

File: move2kube/m2k.sw.yml (hunk 1 above)
Comment (masayag) [reply move2kube/m2k.sw.yml:99]: so their API is outdated?

File: mta/mta.sw.yaml (hunk 2 above)
Comment (masayag) [reply mta/mta.sw.yaml]: It is a nice touch:
Selection_061

File: mta/application.properties [hunk 3]
```diff
@@ -8,3 +8,4 @@ quarkus.rest-client.mta_json.url = ${mta.protocol}://${mta.endpoint}:${mta.port}
 
 #quarkus.log.category.\"org.apache.http\".level = DEBUG
 quarkus.kubernetes-client.devservices.enabled = false
+quarkus.rest-client.notifications_yaml.url=${BACKSTAGE_NOTIFICATIONS_URL:http://host.minikube.internal:7007/api/notifications/}
```
Comment (rgolangh) [inline mta/application.properties:11]: BACKSTAGE_NOTIFICATION is missing 
also the minikube fallback is not relevant

**Comment by gabriel-farache** [comment]
@masayag do we want to merge this now?

File: move2kube/application.properties [hunk 4]
```diff
@@ -8,4 +8,5 @@ mp.messaging.incoming.kogito_incoming_stream.method=POST
 move2kube_url=${MOVE2KUBE_URL:http://move2kube-svc.default.svc.cluster.local:8080}
 quarkus.rest-client.move2kube_yaml.url=${MOVE2KUBE_URL:http://move2kube-svc.default.svc.cluster.local:8080}
 broker_url=${BROKER_URL:http://broker-ingress.knative-eventing.svc.cluster.local/m2k/default}
-quarkus.rest-client.notifications_yaml.url=${BACKSTAGE_NOTIFICATIONS_URL:http://host.minikube.internal:7007/api/notifications/}
\ No newline at end of file
+#quarkus.rest-client.notifications_yaml.url=${BACKSTAGE_NOTIFICATIONS_URL:http://host.minikube.internal:7007/api/notifications/}
+quarkus.rest-client.notifications_yaml.url=https://orchestrator-backstage-orchestrator.apps.cluster-xpxfb.dynamic.redhatworkshops.io/api/notifications/
```
Comment (dmartinol) [inline move2kube/application.properties:12]: I guess one of them is not needed, and the latter points to a specific cluster and it's not configurable

File: mta/application.properties [hunk 5]
```diff
@@ -8,3 +8,5 @@ quarkus.rest-client.mta_json.url = ${mta.protocol}://${mta.endpoint}:${mta.port}
 
 #quarkus.log.category.\"org.apache.http\".level = DEBUG
 quarkus.kubernetes-client.devservices.enabled = false
+#quarkus.rest-client.notifications_yaml.url=${BACKSTAGE_NOTIFICATIONS_URL:http://host.minikube.internal:7007/api/notifications/}
+quarkus.rest-client.notifications_yaml.url=https://orchestrator-backstage-orchestrator.apps.cluster-xpxfb.dynamic.redhatworkshops.io/api/notifications/
```
Comment (dmartinol) [inline mta/application.properties:12]: Same as above

**Comment by masayag** [comment]
I'm going to close this PR. It was a quick one for the sake of the demo. There are proper Jira issues to do the same and fix the other issues. No need to make the effort duplicated.

**Pull request closed without merging**